bin/fastq_dir_to_samplesheet.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import glob
+import argparse
+
+def parse_args(args=None):
+    Description = "Generate nf-core/rnaseq samplesheet from a directory of FastQ files."
+    Epilog = "Example usage: python fastq_dir_to_samplesheet.py <FASTQ_DIR> <SAMPLESHEET_FILE>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FASTQ_DIR", help="Folder containing raw FastQ files.")
+    parser.add_argument("SAMPLESHEET_FILE", help="Output samplesheet file.")
+    parser.add_argument('-st', '--strandedness', type=str, dest="STRANDEDNESS", default='unstranded', help="Value for 'strandedness' in samplesheet. Must be one of 'unstranded', 'forward', 'reverse'.")
+    parser.add_argument('-r1', '--read1_extension', type=str, dest="READ1_EXTENSION", default='_R1_001.fastq.gz', help="File extension for read 1.")
+    parser.add_argument('-r2', '--read2_extension', type=str, dest="READ2_EXTENSION", default='_R2_001.fastq.gz', help="File extension for read 2.")
+    parser.add_argument('-se', '--single_end', dest="SINGLE_END", action='store_true', help="Single-end information will be auto-detected but this option forces paired-end FastQ files to be treated as single-end so only read 1 information is included in the samplesheet.")
```

Comment:
these are super long lines that require me to scroll horizontally during code-review. Consider reformatting using `black`.

Reply:
Oops 👀 Ran `black` locally to fix the formatting https://github.com/nf-core/rnaseq/pull/660/commits/936d73a55850abe1197e35d79495028c7be2e0ea

And created an official issue on the tools repo to add this for linting CI.